Decred Wallet/Extensions/UIColor.swift

```diff
@@ -10,27 +10,43 @@ import Foundation
 import UIKit
 
 extension UIColor {
-    // todo: significantly fewer colors should be left here once the implementation of the new
-    // interface design is completed as the new design uses fewer colors generally.
     struct appColors {
-        // decred colors are from https://decred.org/brand/
-        static let decredBlue = UIColor.init(hex: "#2970FF")
-        static let decredGreen = UIColor.init(hex: "#41BF53")
-        static let decredOrange = UIColor.init(hex: "#ED6D47")
-        
         static let darkYellowWarning = UIColor.init(hex: "#E7C659")
-        static let green = UIColor.init(hex: "#2DD8A3")
-        static let lightGray = UIColor.init(hex: "#DEE1E3")
-        static let lighterGray = UIColor.init(hex: "#C4CBD2")
-        static let offWhite = UIColor(hex:"#F3F5F6")
-        static let lightOffWhite = UIColor(hex: "#F9FBFA")
-        static let yellowWarning = UIColor.init(hex: "#FFC84E")
+        static let _lightGray = UIColor.init(hex: "#DEE1E3") // temporary name, pending when it will be made obsolete and removed
         static let thinGray = UIColor.init(hex: "#a4abb1")
-        static let darkGray = UIColor.init(hex: "#4e5f70")
+        static let darkerGray = UIColor.init(hex: "#4e5f70")
         static let transparentThinGray = UIColor.init(hex: "#a4abb1", alpha: 0.3)
-        static let textGray = UIColor.init(hex: "#596d81")
+        
+        // following are colors from new mockup color guide
+        // ultimately other color constants should end up being obsolete once
+        // the implementation of the new mockup is completed
+        static let lightBlue = UIColor.init(hex: "#2970FF")
+        static let blue = UIColor.init(hex: "#1B41B3")
         static let darkBlue = UIColor.init(hex: "#091440")
-        static let borderGray = UIColor.init(hex: "E6EAED")
+        static let skyBlue = UIColor.init(hex: "#70CBFF")
+        static let lightSkyBlue = UIColor.init(hex: "#E9F8FE")
+        
+        static let decredGreen = UIColor.init(hex: "#41BF53")
```

Comment:
If `decredBlue` changed to `blue`, `decredGreen` should change to green.
```suggestion
		static let green = UIColor.init(hex: "#41BF53")
```